scripts/src/utils.ts

```diff
@@ -4,6 +4,11 @@ const fromProjectRoot = _path.join.bind(path, __dirname, "../../");
 
 export type ServerError = Error & { syscall: string; code: string; };
 
+export type Nothing = null | undefined;
+export function isNothing(obj: any): obj is Nothing {
+	return obj === null || obj === undefined;
```

Comment:
Is this a standard pattern?

Reply:
we can just do:
```
return obj == null;
```
but then someone might change it to `===` and that will ruin it.
this way is more explicit, which i prefer